### PR TITLE
issue-16-specify-dns-server

### PR DESCRIPTION
**Pull request by CarlaLlama**

### CODE REVIEW

**Comment by CarlaLlama** [comment]
@hodgestar what do you think? :)

File: vumi_http_proxy/test/test_queen_of_ni.py [hunk 1]
```diff
@@ -24,11 +25,16 @@ def test_queen_of_ni(self):
...
-    def make_blacklist(self, blacklist):
+    def make_config(self, blacklist, dnsservers):
         filename = self.mktemp()
         with open(filename, 'w') as stream:
             stream.write("proxy-blacklist:\n")
             for ip_addr in blacklist:
                 stream.write(" - " + ip_addr + "\n")
+            stream.write("dns-servers:\n")
+            for server in dnsservers:
+                stream.write(" - " + str(server) + "\n")
```
Comment (hodgestar) [inline vumi_http_proxy/test/test_queen_of_ni.py]: I realised we can use `yaml.safe_dump` to create the string to write.

**Comment by hodgestar** [comment]
Looks great! Woot. :)

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**